angr/analyses/cfg/cfg_fast.py

```diff
@@ -1221,6 +1225,24 @@ def _post_analysis(self):
 
         self._finish_progress()
 
+    def _do_full_xrefs(self):
+        l.info("Building cross-references...")
+        # Time to make our CPU hurt
+        state = self.project.factory.blank_state()
+        for f_addr in self.functions:
+            try:
+                f = self.functions[f_addr]
+                if f.is_simprocedure:
+                    continue
+                l.debug("\tFunction %s" % f.name)
+                # constant prop
+                prop = self.project.analyses.Propagator(func=f, base_state=state)
+                # Collect all the refs
+                self.project.analyses.XRefs(func=f, replacements=prop.replacements)
+            except Exception as e:
+                
+                l.exception("Error collecting XRefs for function %s" % f.name)
```

Comment:
Don't construct message strings. Use `l.exception("Error collecting XRefs for function %s", f.name)` instead.